src/Config/CS_SDK.props

```diff
@@ -63,10 +63,6 @@
   <Target Name="CleanIntermediaryFolder" Condition="Exists('$(ProjectDir)$(BaseIntermediateOutputPath)')" BeforeTargets="Clean" >
      <RemoveDir ContinueOnError="WarnAndContinue" Directories="$(ProjectDir)$(BaseIntermediateOutputPath)" /> <!-- int -->
   </Target>
-  <Target Name="CleanObjFolder" Condition="Exists('$(ProjectDir)obj')" BeforeTargets="Clean">
-	 <RemoveDir ContinueOnError="WarnAndContinue" Directories="$(ProjectDir)obj" />
-	 <!-- obj -->
-  </Target>
```

Comment:
1. Intellisense uses the contents of the obj folder...so it is not a good if this folder is missing
2. The issue we are seeing happens because the obj folder is created when nuget restores the project packages...but the Clean Task is called after that.
We could, in theory, fix it by adding the proper BeforeTargets to also depend on the Nuget restore Target. But I could not find any documentation on it...so I opted to stop doing cleanup manually.
Cleanup is already being done automatically (although I am not sure how well)